lms/djangoapps/projects/views.py

```diff
@@ -320,6 +320,15 @@ class ProjectsViewSet(viewsets.ModelViewSet):
     serializer_class = ProjectSerializer
     model = Project
 
+    def get_queryset(self):
+        target_content_id = self.request.QUERY_PARAMS.get('content_id')
```

Comment:
Because a "project" is essentially a specialized relationship between a course and a piece of content, I think you need to include both the course_id and the content_id in this filter.